### PR TITLE
Managing Connections overhaul - Java

### PR DESCRIPTION
**Pull request by daschl**
Java chapter overhaul on Managing Connections

ping @amarantha-k  / @marijadjovanovic / @simonbasle

### CODE REVIEW

File: content/sdk/java/managing-connections.dita [hunk 1]
```diff
@@ -47,14 +46,40 @@ Bucket bucket2 = cluster.openBucket("bucket2", "password");]]></codeblock>
...
+				not (which is the regular case) it gets also shut down.</p>
+
+<p>After a disconnect, it is not possible to open buckets again, so only use it when you are sure that you do not need access to a <codeph>CouchbaseCluster</codeph> again. If you only want to close a bucket, you can do that without shutting down everything:</p>
+
+<codeblock outputclass="language-java"><![CDATA[Boolean closed = bucket.close();]]></codeblock>
+
+<p>This will release only the resources allocated for this bucket and it is possible to reopen it at
+				a later point.</p>
+
+<p>Especially with <codeph>disconnect</codeph>, make sure you wait until it is finished (so everything is shut down cleanly), either by using the <codeph>toBlocking</codeph> methods or by using other mechanisms in the subscriber (like a <codeph>CountDownLatch</codeph> to orchestrate behavior).</p>
```
Comment (simonbasle) [inline content/sdk/java/managing-connections.dita]: is it still relevant with 2.3? maybe mention that some APIs changed in 2.3?

**Comment by daschl** [comment]
Updated! @amarantha-k @marijadjovanovic please merge :)

**Comment by daschl** [comment]
ping @ingenthr

**Comment by ingenthr** [comment]
Rebased and merged separately with one minor fix.

**Pull request closed without merging**